combined_agent.py

```diff
@@ -0,0 +1,43 @@
+import subprocess
+import waiting
+
+DEFAULT_AGENT_IMAGE = "quay.io/oamizur/assisted-swarm:latest"
+DEFAULT_STDOUT_FILE = "/tmp/combined-agent.stdout"
+DEFAULT_STDERR_FILE = "/tmp/combined-agent.stderr"
+
+class CombinedAgent(object):
```

Comment:
```suggestion
class CombinedAgent:
```

`object` inheritance not needed in Python3

Reply:
OK